README.md

```diff
@@ -25,17 +23,21 @@ The motivation behind this repository is to provide a clear and concise resource
 <a href="https://github.com/benkirk/demo_containers/graphs/contributors">
   <img src="https://contrib.rocks/image?repo=benkirk/demo_containers" />
 </a>
+<a href="https://github.com/ProjectPythia/dask-cookbook/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ProjectPythia/dask-cookbook" />
```

Comment:
I am not sure what would be the best way to handle contributors' list for the Pythia cookbooks for the cookbooks that are adapted from previous materials. This might include my profile twice (but I am okay with that and no need to change)  ;-) 

@jsignell An alternative will be to add your profiles individually or point to a repo that you are the only contributor too. Please see above for how I included Ben...

Reply:
Yeah good point. I kind of like the idea of just relying on the contributors information on github for this repo. I'm not too worried about people showing up twice but eventually if everyone who contributed to the NCAR version has also contributed over here, then we can just have the one contributor list.

Reply:
Thanks @jsignell for your response. In that case, I will leave this as is. 
Overall, this is a bigger issue for the project Pythia maintainers on how to handle the list of contributors for many projects like this, where the cookbooks are adapted from other projects and are collected here as a resource for the community.... 

Maybe a good topic for future Pythia hackathons...

Reply:
I just remembered that there is an authors section as well where there are links to specific individuals, so maybe it already is ok to just rely on github for the list of contributors?